docs/api_summary.rst

```diff
@@ -12,7 +12,7 @@ in *scikit-onnx*.
 Converters
 ==========
 
-.. autofunction:: skl2onnx.convert_sklearn
+.. autofunction:: skl2onnx.to_onnx
```

Comment:
it would be better to discuss with PM to have a better name. In my opinion, convert_ and to_ actually I didn't see too much difference.

Reply:
Some people find this name confusing: they wonder "convert but in which way".

Reply:
I agree with @xadupre ...

Reply:
I drop this PR. It is easier to start from scratch than fixing conflicts. There is no point in renaming if it is not quickly merged.